app/renderer/components/navigation/navigationBar.js

```diff
@@ -115,6 +125,77 @@ class NavigationBar extends React.Component {
       this.props.bookmarked
   }
 
+  // BEM Level: navigationBar__navigationButtonContainer
+  get stopButton () {
+    return <button className={cx({
+      normalizeButton: true,
+      [css(styles.navigationButton, styles.navigationButton_stopButton)]: true
+    })}
+      data-l10n-id='stopButton'
+      onClick={this.onStop}
+    />
+  }
+
+  // BEM Level: navigationBar__navigationButtonContainer
+  get reloadButton () {
+    return <LongPressButton className={cx({
+      normalizeButton: true,
+      [css(styles.navigationButton, styles.navigationButton_reloadButton)]: true
+    })}
+      l10nId='reloadButton'
+      testId='reloadButton'
+      onClick={this.onReload}
+      onLongPress={this.onReloadLongPress}
+    />
+  }
+
+  // BEM Level: navigationBar__navigationButtonContainer
+  get homeButton () {
+    return <button className={cx({
+      normalizeButton: true,
+      [css(styles.navigationButton, styles.navigationButton_homeButton)]: true
+    })}
+      data-test-id='homeButton'
+      data-l10n-id='homeButton'
+      onClick={this.onHome}
+    />
+  }
+
+  // BEM Level: navigationBar
+  get bookmarkButtonContainer () {
+    return <span className={css(
+      commonStyles.navigationBar__buttonContainer,
+      commonStyles.navigationBar__buttonContainer_outsideOfURLbar,
+      styles.navigationBar__buttonContainer_bookmarkButtonContainer
+    )}>
+      <button className={cx({
```

Comment:
btw while I really like the getter way, I think we'd better have our navigator buttons inside its own files. We can separate by "start btn" and "end btn" btw. That is a kind of thing that may change in the near future and tweaks can be made easier. I can help with that, wdyt?